scripts/azure-templates-bootstrapper.yml

```diff
@@ -139,6 +139,8 @@ jobs:
             env:
               JavaSdkDirectory: $(JAVA_HOME)
               LLVM_HOME: $(LLVM_HOME)
+              # There seems to be a bug in some verions of mspdbcmf.exe. This looks to be fixed in a VS preview.
+              AppxSymbolPackageEnabled: false
```

Comment:
This may be because the UWP build system struggles with portable PDB files...
https://github.com/dotnet/sdk/issues/955